lib/cocoapods/installer/xcode/pods_project_generator/aggregate_target_installer.rb

```diff
@@ -6,6 +6,11 @@ class PodsProjectGenerator
         # project and the relative support files.
         #
         class AggregateTargetInstaller < TargetInstaller
+          # @return [AggregateTarget] target
+          #         the target to install
+          #
+          attr_reader :target
```

Comment:
why is this needed? Isnt it available from the base class?

Reply:
was trying to update the type annotation to be `AggregateTarget` for this subclass - is there an easier way to do that?

Reply:
You could use yard's `@!parse` macro: https://www.rubydoc.info/gems/yard/file/docs/Tags.md#parse